packages/toast/index.ts

```diff
@@ -0,0 +1,5 @@
+import toast from 'react-hot-toast'
+import Toaster from './toaster'
+
+export default toast
```

Comment:
Likely a benefit to creating our own facade for toast function with limited support from react-hot-toast defaults. e.g. we don't intend to use toast.success, toast.error, etc.

Reply:
**Default toast:**

```tsx
toast({
  description: "lAliquet in lacus, scelerisque tristique purus. Lob ortis felis orci lacus egestas et turpis.",
  cta: {
    title: 'Standalone link',
    url: '/'
  },
  onDismiss: () => {}
})
```

**Toast types:**

```tsx
toast({
  type: 'language',
  language: 'en',
  description: "lAliquet in lacus, scelerisque tristique purus. Lob ortis felis orci lacus egestas et turpis.",
  cta: {
    title: 'Standalone link',
    url: '/'
  },
  onDismiss: () => {}
})
```

or type specific toast exports that maps closer to notification exports

```tsx
toastWithLanguage({
  language: 'en',
  description: "lAliquet in lacus, scelerisque tristique purus. Lob ortis felis orci lacus egestas et turpis.",
  cta: {
    title: 'Standalone link',
    url: '/'
  },
  onDismiss: () => {}
})
```

Reply:
🤔 I guess we could remove the need to require onDismiss to be defined. Would could handle that within our toast function.

Reply:
```tsx
const toastWithLanguage = ({ language, description, cta }) => {
  return toast((t) => {
    return (
      <NotificationWithLanguage
        language={language}
        description={description}
        cta={cta}
        onDismiss={() => toast.remove(t.id)}
      />
    )
  })
}

toastWithLanguage({
  language: "en"
  description: "lAliquet in lacus, scelerisque tristique purus. Lob ortis felis orci lacus egestas et turpis."
  cta: {
    title: 'Standalone link',
    url: '/'
  }
})
```

Reply:
@alexcarpenter Did you decide on a path forward here, or thinking you'll defer for now?